CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Added first draft of the README
+- During setup, the user now has the ability to configure settings to post messages to a Discord server
+- The project now messages a Discord server when it encounters an unhandled exception if this setting has been configured
```

Comment:
It is generally good to only have 1 bullet point to summarize a single issue/card resolution. If we could condense this into a single point that is a bit more concise, I feel like that would be better overall.

Reply:
I mean, I can, if you want me to, but they are kinda two separate ideas. 
1. Implemented specific functionality
2. Updated existing functionality to use new functionality
I guess I just don't understand why two separate pieces of work should be condensed into one bullet in the changelog when two different things were changed. If this is the case, I feel like we should be doing what the ticket says, no more, no less, which will lead to instances where we forget to update logic in other places that is affected by what we're doing in the moment. 

Lmk what you'd like me to do after hearing these justifications.

Reply:
I feel like this might be an easier discussion to have offline or perhaps even on Slack.

Reply:
I've responded on Slack several times to no avail, so I will continue the discussion here. Is the CHANGELOG supposed to be user- or dev-facing?